test/conformance/runtime/header_test.go

```diff
@@ -91,6 +91,8 @@ func TestShouldHaveHeadersSet(t *testing.T) {
 		// We expect the value to be a 64-bit or 128-bit hex string
 		"x-b3-traceid": regexp.MustCompile("[0-9a-f]{16}|[0-9a-f]{32}"),
 
+		"Traceparent": regexp.MustCompile("[0-9]{2}-[0-9a-f]{32}-[0-9a-f]{16}-[0-9]{2}"),
```

Comment:
traceparent?
It seems everything else is lowercase.

Reply:
Done. seems like we use http healper CanonicalHeaderKey which does this for us